packages/core/src/divider/Divider.css

```diff
@@ -24,4 +24,5 @@
 .saltDivider-vertical {
   width: var(--salt-size-border);
   height: auto;
+  align-self: center;
```

Comment:
Can we review height/width, flex css setting of the Divider? When used within stack layout v.s. not in one. This fix seems to be a temporary fix for this specific scenario.
This could be a symptom of Divider examples are not covered in enough use cases.

Reply:
Approved from Yun